test/Infer/Interpreter/freeze.opt

```diff
@@ -0,0 +1,12 @@
+; RUN: %souper-interpret %solver -input-values=%0=1 %s > %t 2>&1
+; RUN: %FileCheck -check-prefix=TEST1 %s < %t
+; TEST1: Value: 126
+
+; RUN: %souper-interpret %solver -input-values=%0=5 %s > %t 2>&1
+; RUN: %FileCheck -check-prefix=TEST2 %s < %t
+; TEST2-NOT: Value: Poison
```

Comment:
I do not understand this test case: a freeze instruction can never generate poison

Reply:
This is what this test is asserting. Note the "-NOT" after TEST2. "-NOT" means that it never generates poison.

Reply:
ah, got it